query/src/test/java/org/infinispan/query/dsl/embedded/impl/LuceneTransformationTest.java

```diff
@@ -6,58 +6,62 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import org.apache.lucene.search.Sort;
+import org.apache.lucene.search.SortField;
 import org.hibernate.search.engine.search.query.SearchQuery;
 import org.hibernate.search.util.common.SearchException;
-
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.ExpectedException;
-
+import org.infinispan.factories.GlobalComponentRegistry;
+import org.infinispan.manager.EmbeddedCacheManager;
 import org.infinispan.objectfilter.ParsingException;
 import org.infinispan.objectfilter.impl.syntax.parser.IckleParser;
 import org.infinispan.objectfilter.impl.syntax.parser.IckleParsingResult;
 import org.infinispan.objectfilter.impl.syntax.parser.ReflectionEntityNamesResolver;
 import org.infinispan.query.dsl.embedded.impl.model.Employee;
 import org.infinispan.query.helper.SearchMappingHelper;
 import org.infinispan.search.mapper.mapping.SearchMapping;
-
-import org.apache.lucene.search.Sort;
-import org.apache.lucene.search.SortField;
+import org.infinispan.test.SingleCacheManagerTest;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.infinispan.util.concurrent.BlockingManager;
+import org.infinispan.util.concurrent.NonBlockingManager;
+import org.testng.annotations.Test;
 
 /**
  * Test the parsing and transformation of Ickle queries to Lucene queries.
  *
  * @author anistor@redhat.com
  * @since 9.0
  */
-public class LuceneTransformationTest {
+public class LuceneTransformationTest extends SingleCacheManagerTest {
```

Comment:
I ran this test with the changes and the `SearchIndexImpl` was never hit it? Is this test supposed to do any indexing via `SearchIndexImpl`?

Reply:
Good question. It is possible, if no indexing operations are executed. I'm checking it.

Reply:
I checked. The `SearchIndexImpl` is created when the search mapping is created for the test and even if we're not going to index anything the constructor requires the managers to init the flow processor, I think